src/utils/validations/AdrValidation.ts

```diff
@@ -14,102 +14,102 @@ const tc3Types: string[] = [
 
 export const adrValidation = Joi.object().keys({
   vehicleDetails: Joi.object().keys({
-    type: Joi.string().required(),
-    approvalDate: Joi.date().format("YYYY-MM-DD").raw().required()
+    type: Joi.string().required().allow(null, ''),
+    approvalDate: Joi.date().format("YYYY-MM-DD").raw().required().allow(null, '')
   }).required(),
-  permittedDangerousGoods: Joi.array().items(Joi.string()).min(1).required(),
-  compatibilityGroupJ: Joi.boolean().optional().allow(null),
-  additionalExaminerNotes: Joi.string().max(1024).optional().allow(null),
+  permittedDangerousGoods: Joi.array().items(Joi.string()).optional().allow(null, ''),
```

Comment:
Is it not possible to just provide the API with an empty array from VTM side instead of null?